### DISCUSSION TITLE
Extension to semantic segmentation

### DISCUSSION THREAD

**Discussion by saivineethabvns** [Trusted-AI/adversarial-robustness-toolbox#1269] [Q&A] [answered]
Can this toolbox be extended to semantic segmentation also. I'm trying it but getting some errors.
Any help in this matter would be of great help.

Thanks in advance.

**Comment by beat-buesser** [comment, marked as answer]
Hi @saivineetha1998 Yes, ART can be extended to support semantic segmentation. Can you describe your project in more detail?

**Reply by saivineethabvns** [reply]
Hi @beat-buesser, I wanted to apply the adversarial attacks and defence methods to semantic segmentation model using this tool box without writing the code from scratch. Could you suggest how this can be extended. Is there any example notebook for the same?
Thanks in advance.
